chia/_tests/core/mempool/test_mempool_manager.py

```diff
@@ -1985,21 +1984,25 @@ async def send_to_mempool(full_node: FullNodeSimulator, spend_bundle: SpendBundl
     async def fill_mempool_with_test_sbs(
         full_node_api: FullNodeSimulator,
     ) -> List[Tuple[bytes32, SerializedProgram, bytes32]]:
-        puzzle = SerializedProgram.to((1, [[ConditionOpcode.REMARK, b"1" * 12_000]]))
-        ph = puzzle.get_tree_hash()
-        for _ in range(3):
-            await full_node_api.farm_new_transaction_block(FarmNewBlockProtocol(ph))
-        coin_records = await full_node_api.full_node.coin_store.get_coin_records_by_puzzle_hash(False, ph)
-        coins = [cr.coin for cr in coin_records if cr.coin.amount == 250_000_000_000]
+        coins_and_puzzles = []
+        # Create different puzzles to avoid the effects of block compression
+        for i in (1, 2):
+            puzzle = SerializedProgram.to((1, [[ConditionOpcode.REMARK, bytes(str(i) * 12_000, "utf-8")]]))
```

Comment:
```suggestion
            puzzle = SerializedProgram.to((1, [[ConditionOpcode.REMARK, bytes([i] * 12_000]]))
```
it seems simpler to build a bytes buffer directly